config/apache.vhost.sample

```diff
@@ -22,7 +22,7 @@
   #
   # IMPORTANT: Be sure to comment out the DocumentRoot, ProxyPass and
   # ProxyPassReverse directives above.
-  #Redirect permanent / https://%ssl_virtualhost_ip%/
+  #Redirect permanent / https://%ssl_virtualhost_fqdn%/
```

Comment:
Here i agree with @dahorak, it is better we use a similar kind of name everywhere align the names, So I have used %ssl_virtualhost_fqdn% instead of %hostname(fqdn)%